uui-core/hooks/useTableState/useTableState.ts

```diff
@@ -14,20 +13,23 @@ export const useTableState = <TFilter = Record<string, any>>(params: IParams<TFi
     const [tableStateValue, setTableStateValue] = useState<DataTableState>({
         topIndex: 0,
         visibleCount: 40,
-        filter: params.initialFilter ?? parseFilterUrl(),
+        filter: params.initialFilter ?? parseUrlFilter(),
         columnsConfig: getColumnsConfig(params.columns, {}),
+        filtersConfig: parseUrlFiltersConfig() ?? {
+            profileStatus: {
+                isAlwaysVisible: true,
+                isVisible: true,
+                order: getOrderBetween(null, null),
+            },
+        },
+        page: 1,
```

Comment:
we should't have default values for paging, because this hook can be used without pagination